hazelcast/include/hazelcast/client/cluster/impl/ClusterDataSerializerHook.h

```diff
@@ -31,7 +31,9 @@ namespace hazelcast {
                 enum ClusterDataSerializerHook {
                     F_ID = 0,
 
-                    ADDRESS = 1
+                    ADDRESS = 1,
+
+                    VECTOR_CLOCK = 43
```

Comment:
Is this leftover from Vector being IdentifiedDataSerializable ?

Reply:
Good catch.